docs/tools/monitoring.md

```diff
@@ -107,3 +107,38 @@ and send the command:
 Do you need more information on how to debug your application using a sniffer?
 Contact us at <a href="mailto:hello@luos.io">hello@luos.io</a>.
 :::
+
+## Inspector
+
+In order to facilitate the debugging process, Luos offers a more advanced monitoring tool, in order to help you investigate the behavior of your network. In other words, inspector is capable of occupying the information of all the nodes and services in the network, as well as all the messages that are transmitted through it, and finally sending them to any other machine via a serial USB port.
```

Comment:
Shouldn't we use "able" instead of "capable" ? 

I don't understand what you mean by "occupying"

Reply:
capable and able are synonims

Reply:
ok.